dir.props

```diff
@@ -269,11 +269,13 @@
 
     <!-- We add extra binplacing for the test shared framework until we can get hardlinking with the runtime directory working on all platforms -->
     <BinplaceTestSharedFramework Condition="'$(_bc_TargetGroup)' == 'netcoreapp'">true</BinplaceTestSharedFramework>
+    <BinplaceILCInputFolder Condition="'$(_bc_TargetGroup)' == 'uapaot' And '$(BinplaceILCInputFolder)' == ''">true</BinplaceILCInputFolder>
 
     <NETCoreAppTestSharedFxVersion>9.9.9</NETCoreAppTestSharedFxVersion>
     <NETCoreAppTestRootPath>$(BinDir)testhost/$(BuildConfiguration)/</NETCoreAppTestRootPath>
     <NETCoreAppTestHostFxrPath>$(NETCoreAppTestRootPath)host/fxr/$(NETCoreAppTestSharedFxVersion)/</NETCoreAppTestHostFxrPath>
     <NETCoreAppTestSharedFrameworkPath>$(NETCoreAppTestRootPath)shared/Microsoft.NETCore.App/$(NETCoreAppTestSharedFxVersion)/</NETCoreAppTestSharedFrameworkPath>
+    <UAPAotILCInputFolder>$(NETCoreAppTestRootPath)ILCInputFolder</UAPAotILCInputFolder>
```

Comment:
Might consider a name like "ILCFXInputFolder" or "ILCFrameworkInput" or something, since this will lack the rd.xml, the test assembly, and other stuff that is needed to really process.